chart/README.md

```diff
@@ -0,0 +1,51 @@
+# Google Managed Prometheus Helm Chart
+
+This helm chart generate elements for Google Managed Prometheus:
+
+- Namespace
+- PriorityClass
+- ClusterRole
+- ClusterRoleBinding
+- Deployment
+- Service
+- ServiceAccount
+- ConfigMap
+
+### Collecting metrics schema
+
+![collecting metrics](doc/write_metrics.png)
+
+### Retrieving metrics schema
+
+![retrieving metrics](doc/read_metrics.png)
+
+### Google Managed Prometheus CRDs
+
+Please note that CRDs are not installed through this Chart, they should by installed separately as recommended by Helm Community [Separate Charts](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-2-separate-charts)
```

Comment:
I think it would be beneficial to host the `gmp-crds` chart alongside the `gmp` chart. Otherwise, users will need to keep up with the correct CRDs using their own process. I find the upgrade process for `kube-prometheus-stack` to be cumbersome, running the `kubectl apply -f crd.yaml` in every cluster is not ideal. Linkerd has started to follow this process: https://github.com/linkerd/linkerd2/tree/main/charts